src/chigame/users/models.py

```diff
@@ -14,10 +14,10 @@ class User(AbstractUser):
     """
 
     # First and last name do not cover name patterns around the globe
-    name = CharField(_("Name of User"), blank=True, max_length=255)
+    name = models.CharField(_("Name of User"), blank=True, max_length=255)
```

Comment:
What is the underscore doing here? (for both this and the email field)

Is it something about machine name vs human readable display name? 
If so might want to add a comment to this mentioning that

Reply:
actually just saw the `gettext_lazy as _`
read that its for simple translation: https://docs.djangoproject.com/en/4.2/topics/i18n/translation/